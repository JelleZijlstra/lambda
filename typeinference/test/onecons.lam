data foo = F int int in
match F 3 4 with {
	F x y -> (print x; print y)
}
