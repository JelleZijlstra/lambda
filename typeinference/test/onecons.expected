%%stdout%%
3
4
Result: module
	let __result__ = ()
end
%%stderr%%
